.github/workflows/actions.yml

```diff
@@ -34,6 +24,36 @@ jobs:
         with:
           options: --check --diff
           src: .
+      - name: Install flake8, isort
+        run: pip install flake8 isort
+      - name: Lint (flake8)
+        run: flake8
+      - name: Lint (isort)
+        run: isort --check .
+      - name: Start PostgreSQL DB
+        run: docker-compose run --detach db
+      # Run `alembic upgrade head` via docker-compose so that it is within
+      # the network that the DB is also in, reachable via DNS name `db`
+      - name: Migrations
+        run: docker-compose run -e CREATE_ALL_TABLES app alembic upgrade head
+        env:
+          CREATE_ALL_TABLES: false
+      - name: Run tests, generate coverage data
+        run: |
+          docker-compose down && \
+          docker-compose build app && \
+          docker-compose run -e COVERAGE_FILE=/etc/conbench-coverage-dir/.coverage app \
+            coverage run --source conbench \
+              -m pytest -vv -s --durations=20 conbench/tests/
```

Comment:
At the very heart this leaves the test runner invocation unchanged:

```
pytest -vv -s --durations=20 conbench/tests/
```

Before, that was `coverage run --source conbench -m pytest -vv --durations=20 conbench/tests/` -- i.e., the same.